src/assets/_project/_blocks/layout/footer/_site-footer.scss

```diff
@@ -119,8 +119,12 @@
             &.facebook:before {
                 content: "\f082";
             }
-            &.twitter:before {
-                content: "\f081";
+            &.twitter {
+                svg {
```

Comment:
would be nice to have the svg inlined here then per page.
https://stackoverflow.com/questions/10768451/inline-svg-in-css